nucypher/blockchain/eth/actors.py

```diff
@@ -142,14 +142,19 @@ class ContractAdministrator(NucypherTokenActor):
         NucypherTokenDeployer,
     )
 
-    upgradeable_deployer_classes = (
+    dispatched_upgradeable_deployer_classes = (
```

Comment:
I find it clearer a name like `deployer_classes_with_dispatcher`.

Reply:
Will follow up in #1314